README.md

```diff
@@ -51,6 +51,39 @@ documentation.
 3. Create a new tag
 4. Publish a wheel `python setup.py bdist_wheel upload`
 
+## Option 2 - Developing/Testing changes to this theme
+
+If you prefer to stay within github universe and provide changes via pull requests, you can do
+```bash
+#Go to target directory, where you want clone cakephp/docs and cakephp/sphinxtheme into
+#In our case we do it in the users home directory 
+cd ~
+
+#Clone the docs (follow installation guidelines of cakephp/docs)
+git clone git@github.com:cakephp/docs
+cd cake_docs
+
+#Note/recommendation: better work with your own fork of cakephp/docs  
+
+#Install sphinx and other requirements for cakephp/docs
+pip install -r requirments.txt
+
+This will install also cakespinx theme as python distribution package in your /usr/local/lib/<python>/dist-packages/cakephpsphinx. We come in the later step back to it.
+
+#Clone the cakephp sphinx theme
```

Comment:
```suggestion

```bash
#Clone the cakephp sphinx theme
```

Reply:
the ending backticks should be there, you are right
the two sentences on notes should be put under comment as well. ( Even the code will not work by copy/paste because of the link to python package )

i checked the patch and the starting and ending tripple backticks should be in.

Reply:
suggestion added to patch-2 with other small changes - thanks for review, I guess, the instruction is now also more readable